nemo/collections/llm/recipes/__init__.py

```diff
@@ -1,13 +1,14 @@
-from nemo.collections.llm.recipes import llama2_7b, llama3_8b, llama3_8b_16k, llama3_8b_64k, mistral
-from nemo.collections.llm.recipes.log.default import default_log
+from nemo.collections.llm.recipes import llama3_8b, llama3_8b_16k, llama3_8b_64k, llama3_70b, mistral
+from nemo.collections.llm.recipes.log.default import default_log, default_resume
 from nemo.collections.llm.recipes.optim import adam
 
 __all__ = [
     "llama3_8b",
     "llama3_8b_16k",
```

Comment:
Do we need llama3_8b_16? and llama3_8b_64k? 
We should drop them and maybe add only llama3_70b_64k?

Reply:
They are part of the long context recipe so I think we should keep them

Reply:
I can use another separate PR to address this. https://github.com/NVIDIA/NeMo/pull/10140